forms-flow-web/src/constants/applicationComponent.js

```diff
@@ -128,7 +128,7 @@ export const addHiddenApplicationComponent = (form) => {
       customDefaultValue: "const localdata = localStorage.getItem('UserDetails') &&  JSON.parse(localStorage.getItem('UserDetails')); value= localdata?.family_name || '';" 
     },
     { 
-      label: "Current User Role", 
+      label: "Current User Roles", 
```

Comment:
should need to update in backend also 
`forms-flow-ai-ee\forms-flow-api\src\formsflow_api\constants\__init__.py`

`![image](https://github.com/user-attachments/assets/47477b49-0fc4-44af-86c2-02daf09500e7)

Reply:
resolved this comment by this pr https://github.com/AOT-Technologies/forms-flow-ai/pull/2485/files